test/data/cldf/new_minimal/cognatesets.csv

```diff
@@ -0,0 +1,2 @@
+ID,Name,Comment
+I,I,First person singular pronoun as agent of a transitive clause
```

Comment:
This is confusing as entry of cognate set table, it can't distinguish errors where the concepts and cognate sets were confused.